basket/base/forms.py

```diff
@@ -0,0 +1,37 @@
+from django import forms
+from django.core.exceptions import ValidationError
+from django.core.validators import validate_email
+
+
+class EmailListField(forms.Field):
+    def to_python(self, value):
+        """Normalize data to a list of strings."""
+        if not value:
+            return []
+
+        emails = []
+        for email in [v.strip() for v in value.split("\n")]:
```

Comment:
I think it'd be safer to use `value.splitlines()`, though using `split("\n")` and `strip()` is probably roughly equivalent.

Reply:
It doesn't `strip()` but it probably supports other line boundaries, so I'll switch to `splitlines()`. 👍